Examples/S3EventNotifier/README.md

```diff
@@ -0,0 +1,52 @@
+# S3 Event Notifier
+
+This example demonstrates how to create a Lambda that notifies an API of an S3 event in a bucket.
```

Comment:
This example demonstrates how to write a Lambda function that is invoked by an event originating from Amazon S3, such as a file has been created in an S3 bucket.